packages/marko-core/middleware/clean-marko-response.js

```diff
@@ -5,10 +5,10 @@ const stripComments = chunk => chunk.replace(/<!--[A-Z][#/].*?-->/g, '');
 module.exports = ({ enabled = true } = {}) => (req, res, next) => {
   const { write } = res;
   res.write = function clean(...args) {
-    if (!enabled) {
+    const [chunk] = args;
+    if (!enabled || typeof chunk !== 'string') {
```

Comment:
@zarathustra323 - this looks good, although it did confuse me at first.  Might read more cleanly if it was inverted:

`
if (enabled && typeof chunk === 'string') {
      const cleanedArgs = [...args];
      cleanedArgs[0] = stripAppendedJS(stripComponentJS(stripComments(chunk)));
      write.apply(this, cleanedArgs);
    } else {
      write.apply(this, args);
    }
`